android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -84,12 +84,7 @@ class Screen constructor(context: ReactContext?) : FabricEnabledViewGroup(contex
     }
 
     val headerConfig: ScreenStackHeaderConfig?
-        get() {
-            val child = getChildAt(0)
-            return if (child is ScreenStackHeaderConfig) {
-                child
-            } else null
-        }
+        get() = getChildAt(0).takeIf { it is ScreenStackHeaderConfig } as ScreenStackHeaderConfig?
```

Comment:
```suggestion
        get() = getChildAt(0) as? ScreenStackHeaderConfig
```

Wouldn't that give the same result?

Reply:
So I played with this a bit in Kotlin playground and it seems that this is the case.
[7aad6e6](https://github.com/software-mansion/react-native-screens/pull/1558/commits/7aad6e6a32178303c9905760644a2cd748911077)